TradeItIosTicketSDK2/TradeItBalanceService.swift

```diff
@@ -0,0 +1,33 @@
+@objc public class TradeItBalanceService: NSObject {
+    private let session: TradeItSession
+
+    init(session: TradeItSession) {
+        self.session = session
+    }
+
+    func getAccountOverview(
+        _ accountOverviewRequest: TradeItAccountOverviewRequest,
+        onSuccess: @escaping (TradeItAccountOverviewResult) -> Void,
+        onFailure: @escaping (TradeItErrorResult) -> Void
+    ) {
+        accountOverviewRequest.token = self.session.token
+
+        let request = TradeItRequestResultFactory.buildJsonRequest(
+            for: accountOverviewRequest,
+            emsAction: "balance/getAccountOverview",
+            environment: self.session.connector.environment
+        )
+
+        self.session.connector.send(request, targetClassType: TradeItAccountOverviewResult.self) { result in
```

Comment:
This seems to be duplicated multiple times in the server. Might be a nice pattern to extract of:

1. Take data object
2. Construct request object
3. Pass to connector